### DISCUSSION TITLE
Copy/export/import from space to space (pages, groups)

### DISCUSSION THREAD

**Discussion by Gomilu** [GitbookIO/community#21] [5. Export]
### What's your feature/idea?

I would like to copy one page from one space to another space. 

### What's your desired outcome?

Copy one page from one space to another space, or even a lot of pages/groups
A tedious alternative is to command+A, copy, paste to a newly created page within the space, and adding the title as well.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by ajg54** [comment]
This is a basic feature which arguably should have been in the initial specifications. It makes the solution look incomplete and gives the bizarre situation that it is perhaps easier to import content from a completely different system than it is from one of your own spaces 🙃

**Reply by JessicaHerbert** [reply]
I agree with this - I've found myself documenting in Notion first and then importing using markdown into Gitbook.  Ideally I'd love to just do it all in Gitbook

**Comment by JessicaHerbert** [comment]
The ability to move a document from a space to another space in a different collection is a huge feature I'd love to see.   As our organization is using Gitbook as our internal handbook, we are figuring out where things live and often finding ourselves cutting and pasting very often which is less than ideal.

**Comment by OtherToday** [comment]
Hi, how is this feature coming along? I need to merge multiple separate spaces and pages into one wiki and not looking forward to doing it manually!

**Reply by OtherToday** [reply]
Bump

**Comment by Barukimang** [comment]
Totally agree, this feature would make my life sooo easy :)

**Comment by aikoii** [comment]
Thanks all for sharing your thoughts on this! It seems like things are a bit trickier than we thought and at the moment, there isn't any active development happening. But we're definitely keen on making this happen and we'll keep you in the loop if anything changes 🤞.

**Reply by benzitney** [reply]
Is there at least a reliable workaround? I just came to look for a feature request around this to upvote because I'm copy-pasting whole pages between spaces and all of the images are breaking whenever I do, so I keep manually needing to copy over the images one by one and it feels incredibly tedious

**Reply by bigmike36c** [reply]
In the meantime, is there a way to enable a shared media manager so the images are imported properly when you copy and paste a page to a new space? This is the current result:

<img width="883" alt="Screenshot 2023-08-09 at 4 02 09 PM" src="https://github.com/GitbookIO/community/assets/81941044/7a872114-2f92-4240-ab21-45ac77e6388d">

**Reply by aikoii** [reply]
I'm afraid this is currently a limitation we have 😔.  It's best to upload the images to the second space and insert them into the page.

**Reply by bigmike36c** [reply]
Is there a way to batch export and batch import? The articles I'm copying have a ton of images.

**Reply by xavier-chiliz** [reply]
Losing the images when copy-pasting is insanely painstaking, has there been improvement planned around this topics since August 2023 @aikoii ?

**Comment by OtherToday** [comment]
You can download as pdf, then open in desktop version of word then save as
doc x then upload. You loose the formatting and headers convert to H2
instead of H1 but you get the basics

On Tue, Aug 1, 2023 at 6:18 AM benzitney ***@***.***> wrote:

> Is there at least a reliable workaround? I just came to look for a feature
> request around this to upvote because I'm copy-pasting whole pages between
> spaces and all of the images are breaking whenever I do, so I keep manually
> needing to copy over the images one by one and it feels incredibly tedious
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/GitbookIO/discussions/21#discussioncomment-6601572>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUXTXVVCYQMGRYCENEYBVV3XTCGQTANCNFSM6AAAAAAT5X5IIY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Gareth Owen Lloyd
Director
+44 7971186652

www.other.today

**Reply by xavier-chiliz** [reply]
The issue is that exporting as a PDF is limited to the Pro plan, so all other users are stuck with copy-pasting docs and images.

**Comment by kryp70m4v3r1ck** [comment]
same, such a sophisticated platform and the most basic feature isn't integrated... damn

**Reply by rodrcastro** [reply]
Hey @kryp70m4v3r1ck! We hear you, and we are sorry that this is not an option now. Since this thread started, we have improved our copy-and-paste abilities inside the editor. You can now copy and paste entire pages - blocks included - to another page quickly. While still not ideal, it should bridge the gap more easily.

**Reply by bigmike36c** [reply]
Any updates on [enabling a shared media manager](https://github.com/orgs/GitbookIO/discussions/21#discussioncomment-6684240) so images are included in the copy and paste?

**Comment by cekageka** [comment]
This is still a very necessary feature. Especially because the import option is limited to 20 (!) pages - how are we supposed to migrate to Gitbook if we have a third-party wiki with 100+ pages? We can split it into multiple 20-page imports, but without a copy of space2space we can't merge the imports.

Any suggestions?

**Comment by b9Joker108** [comment]
I could really do with this proposed function, I am presently, manually breaking up a large document into sections piecemeal, which is very tedious.

**Comment by Jderoeck** [comment]
Any update on this feature? I am migrating tons of pages to one space to have a fully documented site for my clients. This is taking hours of time because I have to reconnect the links.

**Comment by aikoii** [comment]
Hey folks! ☀️

Great news - you can now copy and paste pages between spaces 🎉

Check out the details in our [changelog](https://gitbook.com/docs/changelog#copy-and-paste-pages-between-spaces-and-change-requests
).
